src/parsing/css-style-value-parsing.js

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//     You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//     See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(internal) {
+
+  var parsers = {};
+  parsers[CSSNumberValue] = internal.parsing.consumeNumber;
+  parsers[CSSLengthValue] = internal.parsing.consumeLength;
+  parsers[CSSTransformValue] = internal.parsing.consumeTransform;
+
+  function consume(consumer, string) {
+    if (!consumer) {
+      return null;
+    }
+    return consumer(string);
+  }
+
+  function consumeKeyword(property, string) {
+    var consumed = internal.parsing.consumeToken(/^[a-z_]+[a-z0-9_-]*/i, string);
```

Comment:
This regex is meant to be a representation of https://www.w3.org/TR/css-syntax-3/#ident-token-diagram

/^ anchor start of string
[a-z_]+ one or more of alphabetical characters or underscore
[a-z0-9_-]* zero or more of alphanumeric, underscore, or dash
/i case insensitive for alphabetical characters